library/modules/bigip_device_auth_radius.py

```diff
@@ -0,0 +1,754 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2018, F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: bigip_device_auth_radius
+short_description: Manage RADIUS authentication on a BIG-IP
+description:
+  - Manage the RADIUS authentication configuration.
+version_added: 2.9
```

Comment:
use template this is incorrect, review the templated files and correct the version